pkg/promtail/targets/windows/targetmanager_windows.go

```diff
@@ -0,0 +1,89 @@
+//+build windows
+
+package windows
+
+import (
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/grafana/loki/pkg/logentry/stages"
+	"github.com/grafana/loki/pkg/promtail/api"
+	"github.com/grafana/loki/pkg/promtail/scrapeconfig"
+	"github.com/grafana/loki/pkg/promtail/targets/target"
+)
+
+// TargetManager manages a series of windows event targets.
+type TargetManager struct {
+	logger  log.Logger
+	targets map[string]*Target
+}
+
+// NewTargetManager creates a new Windows managers.
+func NewTargetManager(
+	logger log.Logger,
+	client api.EntryHandler,
+	scrapeConfigs []scrapeconfig.Config,
+) (*TargetManager, error) {
+
+	tm := &TargetManager{
+		logger:  logger,
+		targets: make(map[string]*Target),
+	}
+
+	for _, cfg := range scrapeConfigs {
+		registerer := prometheus.DefaultRegisterer
```

Comment:
Can you change this to be a registerer that's passed in to NewTargetManager? 🙏

Reply:
done